supervision/detection/line_zone.py

```diff
@@ -84,10 +86,18 @@ def __init__(
                 to consider when deciding on whether the detection
                 has passed the line counter or not. By default, this
                 contains the four corners of the detection's bounding box
+            crossing_acceptance_threshold (int): Detection needs to be seen
```

Comment:
We have similar argument in tracker called `minimum_matching_threshold`, why not try to stay consistent and call it `minimum_crossing_threshold`.